content/platforms/kubernetes/tasks/ldap-on-k8s.md

```diff
@@ -159,10 +159,10 @@ ldap_password: admin
 
 ### Using rladmin to update the configuration
 
-If this information was in a file called “ldap.conf”, you can connect to a Redis Enterprise Node pod and configure the server via:
+If this information was in a file called “ldap.conf” stored in "/tmp", you can connect to a Redis Enterprise Node pod and configure the server via:
 
 ```sh
-rladmin cluster config saslauthd_ldap_conf ldap.conf
+rladmin cluster config saslauthd_ldap_conf /tmp/ldap.conf
```

Comment:
We just merged a PR from @laurentdroin editing this file to `/path/to/ldap.conf`. Should it be specifically mentioned that this is stored in `/tmp` instead?

Reply:
No, I don't think there is any assumption that the file would be in /tmp.
The lines above say:

```
If this information was in a file called “ldap.conf”, you can connect to a Redis Enterprise Node pod and configure the server via:

rladmin cluster config saslauthd_ldap_conf ldap.conf
```

So, no assumption where the file is (IMO).